include/sidekiq/sidekiq_base.h

```diff
@@ -66,19 +66,22 @@ namespace gr {
 		template<typename HdlType>
 		class SIDEKIQ_API sidekiq_base {
 		public:
-			sidekiq_base(int sync_type, HdlType handle_type, gr::sidekiq::sidekiq_functions<HdlType> sidekiq_functions);
+			sidekiq_base(int input_card_number, int port1_handle, HdlType port2_handle, HdlType handle_two, gr::sidekiq::sidekiq_functions<HdlType> sidekiq_functions);
```

Comment:
This  has `int` and `HdlType` used for similar input args.  I think it should probably be one or the other (probably `HdlType` based on other code).  Also, I am not sure what the difference is between `port2_handle` and `handle_two` are?  Maybe `handle_two` should be renamed?

Reply:
Another good catch.